src/extension.ts

```diff
@@ -75,6 +76,7 @@ export async function activate(context: vscode.ExtensionContext): Promise<vscode
 		});
 	});
 	outputChannel.appendLine(`Registered command as part of sfdx-code-analyzer-vscode activation.`);
+	registerAll();
```

Comment:
I see this is inside the activate function. Does this run every time the scanner's action is invoked - or does vs code keep track of things in memory? I ask because registerAll needs to have been invoked to make the unfortunately global state ready for the action to perform properly. If not, then we may need to move this registerAll as the first step of the run method in the scanner.ts file.

Reply:
`activate` runs when the extension is activated. Currently, the events for that are "a project with a `sfdx-project.json` file being opened" and "one of the commands being executed for the first time (if the extension isn't already active)".

Reply:
Josh is correct. As we discussed last week, this is now activated once a project with a sfdx-project.json file (a valid DX project) is loaded. Resolving this comment. Please re-open if there are any other concerns.